yt_dlp/extractor/arte.py

```diff
@@ -217,43 +216,39 @@ class ArteTVPlaylistIE(ArteTVBaseIE):
     _VALID_URL = r'https?://(?:www\.)?arte\.tv/(?P<lang>%s)/videos/(?P<id>RC-\d{6})' % ArteTVBaseIE._ARTE_LANGUAGES
     _TESTS = [{
         'url': 'https://www.arte.tv/en/videos/RC-016954/earn-a-living/',
-        'info_dict': {
-            'id': 'RC-016954',
-            'title': 'Earn a Living',
-            'description': 'md5:d322c55011514b3a7241f7fb80d494c2',
-        },
-        'playlist_mincount': 6,
+        'only_matching': True,
     }, {
         'url': 'https://www.arte.tv/pl/videos/RC-014123/arte-reportage/',
+        # too long to comfortably test
         'only_matching': True,
     }]
 
     def _real_extract(self, url):
         lang, playlist_id = self._match_valid_url(url).groups()
-        collection = self._download_json(
-            '%s/collectionData/%s/%s?source=videos'
+        playlist = self._download_json(
+            '%s/playlist/%s/%s'
             % (self._API_BASE, lang, playlist_id), playlist_id)
```

Comment:
might as well replace this with fstrings

Reply:
@dirkf is going to hate it

Reply:
A triviality, but you could leave the original as a comment to save back-porting effort!